src/components/Nav/Nav.scss

```diff
@@ -0,0 +1,15 @@
+.Nav {
+  &__links {
+    display: flex;
+    justify-content: space-between;
+    margin: 1.5rem;
+
+    .Nav__link {
```

Comment:
This is not BEM at the moment it is BEE as you have an element inside an element 

```scss
// BLOCK
.nav {
// ELEMENT
  &__links {
 // STYLES HERE ...
  }
// ELEMENT
  &__link {
     // STYLES HERE ...
    // MODIFIER 
    &:hover {
     // STYLES HERE ...
    }
  }
}
```